### DISCUSSION TITLE
Is this supposed to happen?

### DISCUSSION THREAD

**Discussion by cosmicgalaxies3** [3kh0/ext-remover#1193] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
This exthang3r does work, but it always kicks me out because of light speed filter agent. Is it because it isn't recommended for light speed or is It meant to be?? Send help please!!

**Comment by S-PScripts** [comment]
probably just doesn't work as well on lightspeed compared to other extensions

**Reply by cosmicgalaxies3** [reply]
OHHH Thx

**Reply by cosmicgalaxies3** [reply]
Another thing is that my sister can use it for as long as she wants with Lightspeed too. Did she use a trick? let me know!

**Reply by ghost** [reply]
I don't think so

**Reply by ghost** [reply]
Maybe its because of yours only

**Reply by cosmicgalaxies3** [reply]
Alr I'll try again sometime later
